src/pages/Scheduling/utils.ts

```diff
@@ -98,14 +98,9 @@ export function computeAppointmentSlots(
       const slotStart = parse(slot.start_time, "HH:mm", referenceDate);
       const slotEnd = parse(slot.end_time, "HH:mm", referenceDate);
       if (
-        isWithinInterval(slotStart, {
-          start: exceptionTime,
-          end: exceptionEndTime,
-        }) ||
-        isWithinInterval(slotEnd, {
-          start: exceptionTime,
-          end: exceptionEndTime,
-        })
+        (slotStart >= exceptionTime && slotStart < exceptionEndTime) ||
+        (slotEnd > exceptionTime && slotEnd <= exceptionEndTime) ||
+        (slotStart <= exceptionTime && slotEnd >= exceptionEndTime)
```

Comment:
can we stick with the same condition logic as how it's specified in backend:

https://github.com/ohcnetwork/care/blob/develop/care/emr/api/viewsets/scheduling/availability.py#L367-L368